### PR TITLE
Lower fixed point operators to corresponding Calyx primitive.

### PR DESCRIPTION
**Pull request by cgyurgyik**
Addresses #356.

### CODE REVIEW

File: src/main/scala/backends/futil/FutilBackend.scala [hunk 1]
```diff
@@ -490,15 +500,15 @@ def emitInvokeDecl(app: EApp)(implicit store: Store, id2FuncDef: FunctionMapping
         val compName = op.op match {
           case "+" => "add"
           case "-" => "sub"
-          case "*" => "mult_pipe"
-          case "/" => "div_pipe"
+          case "*" => "mult"
+          case "/" => "div"
```
Comment (rachitnigam) [inline src/main/scala/backends/futil/FutilBackend.scala:504]: No, this doesn't work. 

The main `div` and `mult` primitives that can actually be turned into hardware have the suffix `_pipe` in the standard library. The fixed point primitive implemented for `div` is not synthesizable hardware. We need an equivalent `fixed_p_div_pipe` primitive to make things work.

Comment (cgyurgyik) [reply src/main/scala/backends/futil/FutilBackend.scala:504]: When not fixed point, It is being emitted with the `_pipe` suffix for multi-cycle binary operations. What am I missing here?

Comment (cgyurgyik) [reply src/main/scala/backends/futil/FutilBackend.scala:504]: As far as creating synthesizable components, created an issue in Calyx:
https://github.com/cucapra/calyx/issues/415
I'll get around to this.

Comment (rachitnigam) [reply src/main/scala/backends/futil/FutilBackend.scala:504]: Here is the problem: `std_mult` and `std_div` are not hardware synthesizable. the `_pipe` variants are which is why Dahlia defaults to it. This is also true of the fixed point variants which use `*` and `/` operators. The correct solution is to remove `std_mult` and `std_div` altogether and just use the synthesizable variants which is what #355 and related PRs are getting at.

A short term solution is renaming the fixedpoint operators in the Calyx library to have `_pipe` in their names.

**Pull request closed without merging**